### ISSUE TITLE
Enhanced laser relay functionality: Allow inputs to raise/lower the relay into the ground, preventing it from being activated.

### ISSUE THREAD

**Issue by Critfish** [BEEmod/BEE2-items#4303]
### Description of the feature

As the title suggests, this is a new feature for the default laser relay item, allowing it to be lowered into the ground via inputs, or starting lowered so it can be raised by an input. When lowered, the relay is impossible to activate.

- Lowered state would be controlled via a new "Start Enabled" widget, with checked = raised and unchecked = lowered.
- If there are no inputs to the relay, it generates as a normal relay.
- If the relay receives inputs, and thus is retractable, it will come with a pair of signages on it to maintain element visibility when the relay is lowered. The signages would be an arrow, pointing to the hole with the relay inside, and a custom signage depicting the retractable relay.
- Because laser relays already take up embed space, this could be implemented seamlessly without breaking existing chambers or introducing a separate retractable relay item.

### Why should this be implemented

This would make laser relays more versatile as a puzzle element, introducing new puzzle design possibilities. In addition, BEE also introduces a similar functionality to Aerial Faith Plates, so there is precedent for this.

### Additional information

![image](https://user-images.githubusercontent.com/58682909/227635113-393a41e5-6a6d-4224-9544-846041966479.png)

**Comment by HeimlichLaboratories** [comment]
Good idea. The sounds should probably vary in each style; I'm thinking a panel moving sound for P2, the apc engine for P1 and one of the many industrial lift sounds for OldAp.
